common-util/AbiAndAddresses/tokenomics.jsx

```diff
@@ -1,8 +1,8 @@
-export const TOKENOMICS_ADDRESS_MAINNET = '0xD599ED4a5c45Fa66AA3C474b00701Fd452a2C30e';
+export const TOKENOMICS_PROXY_ADDRESS_MAINNET = '0xc096362fa6f4A4B1a9ea68b1043416f3381ce300';
 
-export const TOKENOMICS_ADDRESS_GOERLI = '0x93Cd3f6DcE64d67f4420939865A00aC89776D4b5';
+export const TOKENOMICS_ADDRESS_GOERLI = '0x10100e74b7F706222F8A7C0be9FC7Ae1717Ad8B2';
 
-export const TOKENOMICS_ABI_MAINNET = [
+export const TOKENOMICS_ABI = [
```

Comment:
Make sure it matches this: https://github.com/valory-xyz/autonolas-tokenomics/blob/main/abis/0.8.20/Tokenomics.json